validator/keymanager/v2/derived/derived.go

```diff
@@ -288,3 +385,52 @@ func (dr *Keymanager) generateKeystoreFile(privateKey []byte, publicKey []byte,
 	}
 	return json.MarshalIndent(keystoreFile, "", "\t")
 }
+
+func generateDepositTransaction(
```

Comment:
In a follow-up PR, ill abstract this and perform a full cleanup of the accounts v2 code for deduplication